### PR TITLE
Wiki Editor - Disable Live Auto Complete OSF-2096

### PR DESCRIPTION
**Pull request by sbt9uc**
Purpose
--------
To disable Ace Auto Complete so that it does not pop up on the first character of every word that is typed.

Changes
------
Disabled Ace Live Auto complete. Ace Basic Auto Complete is still enabled, which means the feature is still there, but must be prompted each time by the user. 

Side Effects
------
Auto Complete no longer pops up for users that need it. It can still be accessed with 'control' + 'space', but for users that rely heavily on this feature, that can be tedious.

### CODE REVIEW

File: website/addons/wiki/static/WikiEditor.js [hunk 1]
```diff
@@ -26,7 +26,10 @@ ko.bindingHandlers.ace = {
     update: function (element, valueAccessor) {
         var content = editor.getValue();        // Content of ace editor
         var value = ko.unwrap(valueAccessor()); // Value from view model
-
+        editor.setOptions({
```
Comment (abought) [inline website/addons/wiki/static/WikiEditor.js]: Why is autocompletion turned off in the `update` callback instead of in `init`?

**Comment by abought** [comment]
Confirmed that basic autocomplete is available, and that the live feature is turned off.

Can you explore whether putting the deactivation in the `update` callback is really necessary? It may be slightly more performant to do it in `init`, which only fires once.

CR1 complete; no other comments.

**Comment by abought** [comment]
To clarify the above comment, it would be helpful if you could resolve the `init`/`update` question before this actually moves on to CR2.

**Comment by sbt9uc** [comment]
Sounds good, I'll take a look at it.

**Comment by sloria** [comment]
This feature was merged in https://github.com/CenterForOpenScience/osf.io/pull/4763

**Pull request closed without merging**